go.mod

```diff
@@ -1,37 +1,79 @@
 module github.com/invidian/terraform-provider-gpg
 
-go 1.15
+go 1.17
```

Comment:
For darwin-arm64 support, we will be fine bumping it to 1.16, no need to 1.17. 1.16 is still supported, so it should be fine to keep it.

Reply:
Any reason why not use the current version of go? Just curious, because there are no upstream or downstream dependencies that might require backward compatibility.

Reply:
Hmm, yeah. I thought it won't compile with 1.16 when this is set to 1.17, but it seems to work, so it should be fine. Marking as resolved.